###
  例外処理
###

try
  run()
catch err
  console.log err
finally
  quit()

